.gitignore

```diff
@@ -6,4 +6,4 @@ composer.lock
 /website/package-lock.json
 /website/template/assets/*
 !/website/template/assets/.gitignore
-
+.idea
```

Comment:
please revert, and use a global ignore for your personal files (https://stackoverflow.com/a/7335487)

Reply:
Done